td/tracking/templates/tracking/event_form.html

```diff
@@ -53,5 +185,84 @@ <h2 class="panel-title">New Translation Workshop/Event</h2>
 
 {% endblock body%}
 
-{% block extra_script %}
-{% endblock extra_script %}
\ No newline at end of file
+{% block custom_bottom_script %}
```

Comment:
we should avoid inline scripts. my previous would be to put this in it's own module and `require` it in the `main.js` module.